app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<h1 class='text-center mt-5'>Recipes</h1>
+
+<div class="container mt-5">
+  <div class="text-right mb-3 d-flex justify-content-end align-content-center">
+    <%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-success" %>
+  </div>
+
+  <% if @recipes.any? %>
+      <div class="card">
+        <div class="card-body">
+          <div class="table-responsive">
+            <table class="table table-bordered">
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Description</th>
+                  <th>Action</th>
+                </tr>
+              </thead>
+              <tbody>
+                <% @recipes.each do |recipe| %>
+                    <tr>
+                      <td><%= recipe.name %></td>
+                      <td>
+                        <%= recipe.description %>
+                      </td>
+                      <td class="text-center">
+                        <% if recipe.public || current_user == recipe.user %>
+                          <%= link_to "Details", recipe, class: "btn btn-primary btn-sm" %>
+                        <% end %>
+                      </td>
+                      <td class="text-center">
+                        <%= button_to "Remove", recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger btn-sm" %>
+                      </td>
+                    </tr>
+                <% end %>
+              </tbody>
+            </table>
```

Comment:
- [ ] Nice work displaying the recipes :clap: however it would be nice to follow the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true)
Your design :point_down: 
![Screenshot from 2024-01-13 11-33-53](https://github.com/martinkarugaba/recipe_app/assets/91142432/93e9a47e-0035-4bfd-ae49-b51b75b4a1ef)

Reply:
Hi @LionRouge1, we are allowed to style the app the way we choose, as long as we keep the functionality